docs/en/index.rst

```diff
@@ -240,8 +243,8 @@ This shell can take a few different options:
 - ``--queue`` (default: default): Name of queue to bind to
 - ``--processor`` (default: ``null``): Name of processor to bind to
 - ``--logger`` (default: ``stdout``): Name of a configured logger
-- ``--max-iterations`` (default: ``null``): Number of max iterations to run
```

Comment:
I'm ok with this as users will get an error and its a 30s change to fix a broken crontab/config management system. The new name is better :+1:

Reply:
I figured the new name was better. I had a hard time wrapping my head around what an "iteration" was in the context of a queue.